app/src/main/java/xyz/whereuat/whereuat/AutoShareStar.java

```diff
@@ -22,12 +22,12 @@
 
     public AutoShareStar(Context context, AttributeSet attrs) {
         super(context, attrs);
-        TypedArray ta = context.obtainStyledAttributes(attrs, R.styleable.AutoShareStar);
+        TypedArray ta = context.obtainStyledAttributes(attrs, xyz.whereuat.whereuat.R.styleable.AutoShareStar);
```

Comment:
Why do we need to prefix this with the package name? Are you sure we need to do this?